src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write code here
+const allLi = document.querySelector('.tree').querySelectorAll('li');
+
+for (const li of allLi) {
+  if (li.children.length !== 0) {
```

Comment:
The condition 'li.children.length !== 0' might not work as expected if 'li.children' returns an array-like object. It's better to use the 'Array.prototype.some' method to check if there are any children. For example, you could use: 'Array.from(li.children).some(child => child.tagName === 'UL')'.